docs/rl-algorithms/rpo.md

```diff
@@ -0,0 +1,228 @@
+        | dm_control/quadruped-run-v0           | 165.70 ± 43.21                                               | 375.78 ± 119.95                               |
+        | dm_control/quadruped-escape-v0        | 23.82 ± 10.82                                                | 68.19 ± 30.69                                 |
+        | dm_control/quadruped-fetch-v0         | 183.03 ± 37.50                                               | 227.47 ± 21.67                                |
+        | dm_control/reacher-easy-v0            | 769.10 ± 48.35                                               | 740.93 ± 51.58                                |
+        | dm_control/reacher-hard-v0            | 636.89 ± 77.68                                               | 584.33 ± 43.76                                |
+        | dm_control/stacker-stack_2-v0         | 61.73 ± 17.34                                                | 64.32 ± 7.15                                  |
+        | dm_control/stacker-stack_4-v0         | 71.68 ± 29.03                                                | 53.78 ± 20.09                                 |
+        | dm_control/swimmer-swimmer6-v0        | 152.82 ± 30.77                                               | 166.89 ± 14.49                                |
+        | dm_control/swimmer-swimmer15-v0       | 148.59 ± 22.63                                               | 153.45 ± 17.12                                |
+        | dm_control/walker-stand-v0            | 453.73 ± 217.72                                              | 736.72 ± 145.16                               |
+        | dm_control/walker-walk-v0             | 308.91 ± 87.85                                               | 785.61 ± 133.29                               |
+        | dm_control/walker-run-v0              | 123.98 ± 91.34                                               | 384.08 ± 123.89                               |
+
+        Learning curves:
+        ![](../rpo/dm_control_all_ppo_rpo_8M.png)
+    
+    
+    === "MuJoCo v4"
+
+        |                     | ppo_continuous_action_8M ({'tag': ['v1.0.0-13-gcbd83f6']})   | rpo_continuous_action ({'tag': ['pr-331']})   |
+        |:--------------------|:-------------------------------------------------------------|:----------------------------------------------|
+        | HumanoidStandup-v4  | 107841.91 ± 15232.17                                         | 150129.37 ± 6447.36                           |
+        | Humanoid-v4         | 590.33 ± 29.38                                               | 763.79 ± 137.91                               |
+        | InvertedPendulum-v4 | 883.61 ± 25.01                                               | 875.56 ± 39.68                                |
+        | Walker2d-v4         | 2872.50 ± 683.82                                             | 3707.25 ± 346.22                              |
+
+        Learning curves:
+        ![](../rpo/mujoco_v4_part1.png)
```

Comment:
At the end of the learning curves, you should also add the tracked experiments. When invoking the `openrlbenchmark` command, you can add a `--report` to generate a report. Then you can clone this report to the [`openrlbenchmark/openrlbenchmark` project](https://wandb.ai/openrlbenchmark/openrlbenchmark/reportlist). Finally you can embed the report as an iframe in the docs, as done in the docs. The report allows the readers to track where the experiments come from and they can query additional info they are interested (e.g., hardware usage)

Reply:
This is such an awesome tool you develop. So convenient to generate results for tables, curves, and reports from wandb. Awesome!!!

I will add them accordingly.